src/platforms/apple/common/dsym.mdx

```diff
@@ -6,17 +6,14 @@ title: Uploading Debug Symbols
 tags: []
 ---
 
-Sentry requires a dSYM upload to symbolicate your crash logs. The symbolication process unscrambles Apple’s crash logs to reveal the function, file names, and line numbers of the crash. Upload the dSYM file using either [sentry-cli](https://github.com/getsentry/sentry-cli), the [Fastlane](https://fastlane.tools/) action, or set up the Sentry [App Store Connect](#bitcode-appstore) integration for Bitcode builds.
+Sentry requires a dSYM upload to symbolicate your crash logs. The symbolication process unscrambles Apple’s crash logs to reveal the function, file names, and line numbers of the crash. Upload the dSYM file using either [sentry-cli](https://github.com/getsentry/sentry-cli), the [Fastlane](https://fastlane.tools/) action, or set up the Sentry [App Store Connect](#bitcode-appstore) integration for Bitcode builds. Every solution requires a Sentry Auth Token with the [correct permissions](/product/cli/dif/#permissions). You can [create one here](https://sentry.io/api/).
 
-## With Bitcode {#dsym-with-bitcode}
+# With Bitcode {#dsym-with-bitcode}
```

Comment:
It's not a good idea to use an H1 within the page because the title is already considered the H1, so it causes the RH sidebar menu to be rendered improperly (see this heading and **Without Bitcode** no longer show up). It looks like you want to make it a bit easier to find the options under **App Store Connect** so I've suggested an edit in that section to help with that. Please change the heading levels back to what they were previously for this page. 

```suggestion
## With Bitcode {#dsym-with-bitcode}
```